src/pages/create/tabs/ReviewDeployTab/FundingAttributes.tsx

```diff
@@ -281,6 +281,28 @@ export function AllowMintingStatistic({
   )
 }
 
+export function AllowSetTerminalsStatistic({
```

Comment:
Adds this

<img width="1121" alt="Screen Shot 2022-07-05 at 10 49 15 AM" src="https://user-images.githubusercontent.com/94939382/177231352-3b68b870-1676-4302-8702-4ae2b239a3d1.png">